chatgpt.js

```diff
@@ -677,22 +677,23 @@ const chatgpt = { // eslint-disable-line no-redeclare
         return this[targetFuncName](); // call found function
     },
 
-    getAccessToken() {
+    async getAccessToken() {
+        if (Object.keys(chatgpt.openAIaccessToken).length > 0 && 
+            (Date.parse(chatgpt.openAIaccessToken.expireDate) - Date.parse(new Date()) >= 0)) {
+            return chatgpt.openAIaccessToken.token;
+        }
```

Comment:
The original was more readable because the final `return` was not aligned with any condition, can you push it 4-spaces deeper then? Also restore the `// not expired` comment for 2nd condition